app/src/main/java/com/jetpack/compose/learning/viewpager/HorizontalPagerWithFlingBehavior.kt

```diff
@@ -0,0 +1,129 @@
+package com.jetpack.compose.learning.viewpager
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.gestures.FlingBehavior
+import androidx.compose.foundation.gestures.ScrollScope
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.PagerDefaults
+import com.google.accompanist.pager.rememberPagerState
+import com.jetpack.compose.learning.R
+import com.jetpack.compose.learning.theme.AppThemeState
+import com.jetpack.compose.learning.theme.BaseView
+import com.jetpack.compose.learning.theme.SystemUiController
+import dev.chrisbanes.snapper.ExperimentalSnapperApi
+
+class HorizontalPagerWithFlingBehavior : ComponentActivity() {
```

Comment:
Class description is needed. Please mention the description in brief what exctly we are including in this class.